x-pack/plugins/infra/server/lib/alerting/metric_threshold/lib/metric_query.ts

```diff
@@ -77,8 +75,8 @@ export const getElasticsearchMetricQuery = (
             date_range: {
               field: timefield,
               ranges: Array.from(Array(Math.floor((to - from) / intervalAsMS)), (_, i) => ({
-                from: from + intervalAsMS * i + offsetInMS,
-                to: from + intervalAsMS * (i + 1) + offsetInMS,
+                from: from + intervalAsMS * i,
+                to: from + intervalAsMS * (i + 1),
               })),
```

Comment:
Do we understand the reason for getting `* i` ranges instead of just the one range that we need?

Reply:
Likely another holdover from the `date_histogram` style. We only look at the last range when evaluating if the alert should fire or not. @Zacqary any insight here?

Reply:
This was part of partial bucket dropping, under the assumption that if we got 5 buckets then at least one of them wouldn't be a partial. Except the offset was already avoiding partial buckets so I think this is actually just redundant.

Reply:
Removed the buckets, now we only get the last bucket.